### ISSUE TITLE
Opening generate command

### ISSUE THREAD

**Issue by ItadakimasuChan** [SilentNightSound/Genshin-Model-Override#2]
When I run genshin_3dmigoto_generate.py, it instantly closes. I'm assuming that's not suppose to happen?

**Comment by SilentNightSound** [comment]
No, it should be generating the Mod folder for the character. Make sure you are running it like `python genshin_3dmigoto_generate.py -n Character` where Character is the name of the Character folder you are trying to convert to generate the mod from. 

The script should be in the same directory as the folder (i.e. one up from the .vb and .ib files), and you need hash_info.json in the directory as the script as well (the repo has the correct format already setup). If the script encounters any errors, it should print them to the console when it is run.

**Comment by ItadakimasuChan** [comment]
Alrighty, I got it work. Had to change directory in cmd
